public/app/features/manage-dashboards/state/RecentlyDeletedActions.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import { Button } from '@grafana/ui';
+
+import appEvents from '../../../core/app_events';
+import { Trans } from '../../../core/internationalization';
+import { useDispatch } from '../../../types';
+import { ShowModalReactEvent } from '../../../types/events';
+import { useRestoreDashboardMutation } from '../../browse-dashboards/api/browseDashboardsAPI';
+import { setAllSelection, useActionSelectionState } from '../../browse-dashboards/state';
+import { RestoreModal } from '../components/RestoreModal';
+import { useRecentlyDeletedStateManager } from '../utils/useRecentlyDeletedStateManager';
+
+export function RecentlyDeletedActions() {
+  const dispatch = useDispatch();
+  const selectedItems = useActionSelectionState();
+  const [_, stateManager] = useRecentlyDeletedStateManager();
```

Comment:
```suggestion
  const [, stateManager] = useRecentlyDeletedStateManager();
```
Nit: I think you can omit the first return value here if you don't need it (using underscore will still create a variable named `_`).